apps/web/src/components/admin/toggles/RegistrationSettings.tsx

```diff
@@ -116,6 +131,18 @@ export function RegistrationToggles({
 							}}
 						/>
 					</div>
+					<div className="flex items-center py-4 border-t-muted border-t border-b">
```

Comment:
There needs to be a border-b-muted Tailwind class here in order to get it to work with the theme.